app.js

```diff
@@ -62,64 +62,55 @@ if(process.env.VCAP_SERVICES){
-router.get('/config/countBookings', routes.countBookings);
-router.get('/config/countCustomers', routes.countCustomer);
-router.get('/config/countSessions', routes.countCustomerSessions);
-router.get('/config/countFlights', routes.countFlights);
-router.get('/config/countFlightSegments', routes.countFlightSegments);
-router.get('/config/countAirports' , routes.countAirports);
-//router.get('/loaddb', startLoadDatabase);
-router.get('/loader/load', startLoadDatabase);
-router.get('/loader/query', loader.getNumConfiguredCustomers);
-router.get('/checkstatus', checkStatus);
+var schema = require('./routes/schema.js');
+
+// Setup fastify
+var fastify = Fastify({ logger: settings.useDevLogger ? true : false }) // log every request to the console in development
+
+fastify.register(require('fastify-static'), {
+  root: path.join(__dirname, 'public') // set the static files location /public/img will be /img for users
+});
+
+fastify.register(require('fastify-cookie'))
+fastify.register(require('fastify-formbody'))
+
+function router (fastify, opts, next) {
+	fastify.post('/login', { schema: schema.login }, login);
+	fastify.get('/login/logout', { schema: schema.logout }, logout);
+	fastify.post('/flights/queryflights', { schema: schema.queryFlights, beforeHandler: routes.checkForValidSessionCookie }, routes.queryflights);
+	fastify.post('/bookings/bookflights', { schema: schema.bookFlights, beforeHandler: routes.checkForValidSessionCookie }, routes.bookflights);
+	fastify.post('/bookings/cancelbooking', { schema: schema.cancelBooking, beforeHandler: routes.checkForValidSessionCookie }, routes.cancelBooking);
+	fastify.get('/bookings/byuser/:user', { schema: schema.bookingsByUser, beforeHandler: routes.checkForValidSessionCookie }, routes.bookingsByUser);
+	fastify.get('/customer/byid/:user', { schema: schema.getCustomerById, beforeHandler: routes.checkForValidSessionCookie }, routes.getCustomerById);
+	fastify.post('/customer/byid/:user', { schema: schema.putCustomerById, beforeHandler: routes.checkForValidSessionCookie }, routes.putCustomerById);
+	fastify.get('/config/runtime', {}, routes.getRuntimeInfo);
+	fastify.get('/config/dataServices', {}, routes.getDataServiceInfo);
+	fastify.get('/config/activeDataService', {}, routes.getActiveDataServiceInfo);
+	fastify.get('/config/countBookings', {}, routes.countBookings);
+	fastify.get('/config/countCustomers', {}, routes.countCustomer);
+	fastify.get('/config/countSessions', {}, routes.countCustomerSessions);
+	fastify.get('/config/countFlights', {}, routes.countFlights);
+	fastify.get('/config/countFlightSegments', {}, routes.countFlightSegments);
+	fastify.get('/config/countAirports', {}, routes.countAirports);
+	// fastify.get('/loaddb', startLoadDatabase);
+	fastify.get('/loader/load', {}, startLoadDatabase);
+	fastify.get('/loader/query', {}, loader.getNumConfiguredCustomers);
+	fastify.get('/checkstatus', {}, checkStatus);
+
+	next()
+}
```

Comment:
The indentation of your editor uses both spaces and tabs. You might want to run everything through eslint or standard (or semistandard).